### DISCUSSION TITLE
Docker Image not running. Error Command failed with exit code 127

### DISCUSSION THREAD

**Discussion by Zulelee** [FlowiseAI/Flowise#900] [Q&A]
I have been trying to create a docker image of Flowise. I am stuck on this task since 1 week and I can not find the solution. I have searched everything I can regarding it. 

The image was created successfully. But when I run the container with the image, I get the following error:

```
2023-09-09 21:05:08 yarn run v1.22.19
2023-09-09 21:05:08 $ run-script-os
2023-09-09 21:05:08 $ cd packages/server/bin && ./run start
2023-09-09 21:05:09 info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
': No such file or directory't execute 'node
2023-09-09 21:05:09 error Command failed with exit code 127.
2023-09-09 21:05:09 error Command failed with exit code 127.
2023-09-09 21:05:09 info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
``` 
Can anyone please guide me how to solve it?

**Comment by chungyau97** [comment]
Hey @Zulelee, 

I'm able to build and run it on Linux but not in Windows

cc: @HenryHengZJ

**Reply by Zulelee** [reply]
Did you just follow the guidelines by Flowise in Linux?

**Reply by chungyau97** [reply]
Yup, I'm doing it on WSL.

**WSL pass**
![image](https://github.com/FlowiseAI/Flowise/assets/33013947/b4e07037-2861-4823-befa-a45c26e02c75)
![image](https://github.com/FlowiseAI/Flowise/assets/33013947/f9fcb47d-9a64-4437-a8e5-c91ab255949e)

**Window fail**
![image](https://github.com/FlowiseAI/Flowise/assets/33013947/fd082abb-fe10-43db-9177-18a2cbf8c7ae)
![image](https://github.com/FlowiseAI/Flowise/assets/33013947/470c7c4a-c2d1-4529-8fda-88f8da07dada)

**Reply by Zulelee** [reply]
Okay, thanks!

**Comment by sailarg** [comment]
i have the same issue

**Comment by automaton82** [comment]
I submitted a pull request #1054 for the linked issue #911 with a suggested fix to replace `CRLF` line endings on `dev` and `run` files in the `bin` folder with `LF`. This solves it for me on Windows.
